### PR TITLE
Document recent merge in regards to select fields for export screen a…

### PR DESCRIPTION
**Pull request by seamuslee001**
…nd add in the notice about WordPress compatability issue

Overview
----------------------------------------
Documents recent merge #17444 and also puts in a notice about the compatibility change for WordPress

ping @eileenmcnaughton @agh1 @totten @kcristiano @alifrumin

### CODE REVIEW

File: release-notes/5.26.0.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ Released June 3, 2020
 | Introduce features?                                             | **yes** |
 | Fix bugs?                                                       | **yes** |
 
+## NOTICE: WordPress Service URLs
+
+The upcoming release of WordPress 5.5 (ETA July 2016) requires changes to several CiviCRM URLs. CiviCRM 5.26+ adopts a new URL convention. For old URLs, there is transitional support that will work for most web-based visitors. However, this may not be sufficient for *external service integrations* (e.g. certain payment processors or email relays which send notifications to CiviCRM).
```
Comment (kcristiano) [inline release-notes/5.26.0.md:26]: Can we change July 5th to August 11th?  Based on the WP 5.5 roadmap the target is August 11th

Comment (mattwire) [reply release-notes/5.26.0.md:26]: And maybe not 2016 ;-)

Comment (totten) [reply release-notes/5.26.0.md]: Or we could just rollback everything to 2016.

**Comment by seamuslee001** [comment]
Going to merge this now

**Pull request merged**